neutrinoapp/src/Search.jsx

```diff
@@ -155,19 +161,21 @@ export default function Search () {
                     </Link>{}
                   </td>
                 </tr>
-              )
+              ) 
             }
           </tbody>
         </table>
         <div className={classes.root}>
-          <Pagination
-            count={pageCount}
-            size='large'
-            page={page}
-            variant='outlined'
-            shape='rounded'
-            onChange={handleChange}
-          />
+        { error400 ? <p></p> : displayWiki.length < 1 ? <p>No data found for {keyword}</p> : 
```

Comment:
It seems odd to have logic in the return, rather than it being abstracted out somehow? 

However, if this is the best way to do things, then cool, I learned something :)